examples/hawkbit-example-mgmt-simulator/src/main/java/org/eclipse/hawkbit/mgmt/client/scenarios/ConfigurableScenario.java

```diff
@@ -316,31 +316,38 @@ private void waitUntilRolloutIsReady(final Long id) {
 
     private void createDistributionSets(final Scenario scenario) {
         LOGGER.info("Creating {} distribution sets", scenario.getDistributionSets());
+        final int pages = (scenario.getDistributionSets() / PAGE_SIZE) >= 1
```

Comment:
I'm afraid it is still broken. Use `BigDecimal` instead if int or float for the calculation. 

Example:
`new BigDecimal(1002).divide(new BigDecimal(500), BigDecimal.ROUND_UP)` => 3